scripts.d/20_install_autorun.sh

```diff
@@ -10,3 +15,8 @@ Restart=no
 
 [Install]
 WantedBy=multi-user.target
+
+EOF
+
+mkdir -p "mnt/img_root/etc/systemd/system/multi-user.target"
```

Comment:
"multi-user.target" should be "multi-user.target.wants"

push needs to be made before PR is merged